.circleci/code-sniff.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+BUILD_DIR=$(pwd)
+
+# Add WordPress coding standards path to code sniffer
+./vendor/bin/phpcs --config-set installed_paths $BUILD_DIR/vendor/wp-coding-standards/wpcs
+
+if [ ! -d ./web/wp-content ]
+then
+	mkdir ./web/wp-content
```

Comment:
What about a `.gitkeep` file in `/web/wp-content` rather than having to make it? Might preserving that directory make our example repo more comprehensible? Or maybe a README saying something like "Add your custom plugins here?"